src/cli/args.rs

```diff
@@ -145,6 +163,11 @@ pub trait ArgBuilder {
     fn aggregate_private_args<'a>() -> Arg<'a>;
     fn parse_aggregate_private_args(matches: &ArgMatches) -> Vec<Vec<u64>>;
 
+    fn aggregate_context_in_args<'a>() -> Arg<'a>;
```

Comment:
Remove it, we can't support context in this aggregator. It will be handled by an ad-hoc aggregator